content.js

```diff
@@ -1,54 +1,75 @@
 // variables
 let recording = false;
-let count=0, maxCount=0;
+let count = 0,
+  maxCount = 0;
+let minValue = null,
+  maxValue = null;
 
 /**
  * Function to track the users mouse movements subject to the maxCount
  * @param event => For recording the cursor position
  */
 function sendCursorCoordinates(event) {
-    if(count < maxCount) {
-        chrome.runtime.sendMessage({
-            x: event.pageX,
-            y: event.pageY
-        });
-        count += 2;
-    } else {
-        stopRecording();
+  if (count < maxCount) {
+    let x = event.pageX;
+    let y = event.pageY;
+
+    // Apply min and max value constraints
+    if (minValue !== null) {
+      x = Math.max(x, minValue);
+      y = Math.max(y, minValue);
+    }
+    if (maxValue !== null) {
```

Comment:
### Possible Remedies
1. One way of doing it would be to completely ignore the coordinate if it falls out of range

2. The other would be to normalize that coordinate statistically to make it fall inside the allowed range

The first approach better preserves randomness, since mapping/cramming huge numbers inside a small range can lead to loss in degree of randomness. In that case we'll also have to decrease the count variable (number of coords recorded) accordingly.